hydra-node/src/Hydra/Options.hs

```diff
@@ -407,6 +407,8 @@ data BlockfrostChainConfig = BlockfrostChainConfig
   -- ^ Path to the blockfrost project file
   , cardanoSigningKey :: FilePath
   -- ^ Path to the cardano signing key of the internal wallet.
+  , hydraScriptsTxId :: [TxId]
+  -- ^ Identifier of transaction holding the hydra scripts to use.
```

Comment:
Must: not replicate command line options

I think I demonstrated two weeks ago, that command line options like `--cardano-signing-key` and `--hydra-scripts-tx-id` are common to both, `Direct` and `Blockfrost` chain backends. Hence, we should keep them not in `Direct` or `Blockfrost` specific data structures. This will directly drive function structure and code re-use.